src/components/home/displayCharacters.js

```diff
@@ -0,0 +1,234 @@
+          });
+      };
+
+      retrieveComment();
+
+      // postNewComment();
+      insideBtn.addEventListener('click', () => {
+        while (newComment.hasChildNodes()) {
+          newComment.removeChild(newComment.firstChild);
+        }
+
+        const updatePost = {
+          // inputs para el post
+          item_id: idDinamic,
+          username: input1.value,
+          comment: input2.value,
+        };
+
+        const optionsPost = {
+          // FETCH POST COMENTARIOSss
+          method: 'POST',
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify(updatePost),
+        };
+
+        const postNewComment = async () => await fetch(idPath, optionsPost);
+        postNewComment();
+        setTimeout(retrieveComment, 1000);
+      });
+    });
+  });
+
+  close.addEventListener('click', () => {
+    window.location.reload();
+  });
+  await displayLikes();
+  passLike();
+
+  // section reservations
+  const closeRes = document.querySelector('.close__but');
+  const containerRes = document.querySelector('.main__container');
+  const btnReserv = document.querySelectorAll('.reservationsBtn');
+  const nameSpaceRes = document.querySelector('.name__space__reserv ');
+  const photoRes = document.querySelector('.photo__reserv');
+  const contOneRes = document.querySelector('.one_one');
+  const contTwoRes = document.querySelector('.two_two');
+  const contThreeRes = document.querySelector('.three_three');
+  const contFourRes = document.querySelector('.four_four');
+  const addCommentRes = document.querySelector('.name__add_reserv');
+  const input1Res = document.querySelector('.text__main');
+  const input2Res = document.querySelector('.main__mail');
+  const input3Res = document.querySelector('.end__name');
+  const insideBtnRes = document.createElement('button');
+  insideBtnRes.classList.add('inside__button_reserv');
+
+  const newCommentReser = document.createElement('div');
+  newCommentReser.classList.add('newCommentReser');
+  const generalRes = document.querySelector('.main__container');
+  const newParagraphReser = document.createElement('p');
+  newParagraphReser.classList.add('text__paragraph_reser');
+  newCommentReser.appendChild(newParagraphReser);
+  generalRes.insertBefore(newCommentReser, addCommentRes);
+
+  btnReserv.forEach((btn) => {
+    btn.addEventListener('click', () => {
+      const inputMainRes = document.querySelector('.input__main_reserv');
+      const idDinamic = btn.parentElement.id; // id dinamico get
+      const idPath =
+        'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/wHDqf2FyYMmzmK7MMxf9/reservations/'; // id dinamico post
+      const pathGet = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments/?item_id=${idDinamic}`;
+
+      generalRes.style.display = 'block';
+
+      fetch('https://www.breakingbadapi.com/api/characters?limit=12&offset=0') // GET BREAKING BAD DATA
```

Comment:
- [ ] when fetching/posting data using `fetch` we should always use async and await, The keyword async is used to make a function asynchronous. The await keyword will ask the execution to wait until the defined task gets executed. and since we need to wait for the response kindly make sure to use async & await for all of your fetch methods in passLike.js, displayCharacters.js & index.js